README.md

```diff
@@ -1 +1,259 @@
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Set up a "Hello Microverse" project <a name="about-project"></a>
+
+
+**Set up a "Hello Microverse" project** is my first project on Microverse Program, It is focused on learning some important issues that will be used along all program experience.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+<details>
+  <summary>GitHub flow</summary>
+  <ul>
+    <li>GitHub flow</li>
+  </ul>
+</details>
+
+<details>
+  <summary>Commit messages</summary>
+  <ul>
+    <li>Commitmessages</li>
+  </ul>
+</details>
+
+<details>
+<summary>Pull Requests</summary>
+  <ul>
+    <li>Pull Requests</li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+
+- **Creating branches on repositories.**
+- **Use of linter on html&css project.**
+- **Write a descriptive README file.**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+
+- [Live Demo Link](https://yourdeployedapplicationlink.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- **Understand the concept of version control.**
+- **Follow GitHub flow.**
+- **Open a pull request on GitHub.**
+
+<!--
+Example command:
+
+```sh
+ gem install rails
+```
+ -->
+
+### Setup
+
+Clone this repository to your desired folder:
+
+<!--
+Example commands:
+
+```sh
+  cd my-folder
+  git clone git@github.com:myaccount/my-project.git
+```
+--->
+
+### Install
+
+Install this project with:
+
+<!--
+Example command:
+
+```sh
+  cd my-project
+  gem install
+```
+--->
+
+### Usage
+
+To run the project, execute the following command:
+
+<!--
+Example command:
+
+```sh
+  rails server
+```
+--->
+
+### Run tests
+
+To run tests, run the following command:
+
+<!--
+Example command:
+
+```sh
+  bin/rails test test/models/article_test.rb
+```
+--->
+
+### Deployment
+
+You can deploy this project using:
+
+<!--
+Example:
+
+```sh
+
+```
+ -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+
+
+👤 **Author1**
+
+- GitHub: [@oovillagran](https://github.com/githubhandle)
+- Twitter: [@oovillagran](https://twitter.com/twitterhandle)
+- LinkedIn: [linkedin.com/in/oscar-villagran-91b13859](https://linkedin.com/in/linkedinhandle)
```

Comment:
It's great to have these linking 🔗  to your socials, but the links aren't working for now because the markdown isn't correct. This is the fixed version:
```
👤 **Oscar Villagran**

- GitHub: [@oovillagran](https://github.com/goovillagran)
- Twitter: [@oovillagran](https://twitter.com/oovillagran)
- LinkedIn: [linkedin.com/in/oscar-villagran-91b13859](https://linkedin.com/in/loscar-villagran-91b13859)
```